NEWS.md

```diff
@@ -1,6 +1,8 @@
 # serocalculator (development version)
 
 ## New features
+* Change default pipe setting
```

Comment:
shouldn't this go under the "Internal changes" heading?

Reply:
and add the PR # in parentheses: `(#312)`

Reply:
This is well-noted.